### DISCUSSION TITLE
does mcu protocol can be used as output?

### DISCUSSION THREAD

**Discussion by benwadub** [tttapa/Control-Surface#615] [Q&A]
hi, I already managed to use mcu implementation as input but can t make it work as output on button wired on mux
cpp``
NoteValue mute[] = { 
  {mux5.pin(0), {MCU::MUTE_1}},
  {mux5.pin(1), {MCU::MUTE_2}},
  {mux5.pin(2), {MCU::MUTE_3}},
  {mux5.pin(3), {MCU::MUTE_4}},
  {mux5.pin(4), {MCU::MUTE_5}},
  {mux5.pin(5), {MCU::MUTE_6}},
  {mux5.pin(6), {MCU::MUTE_7}},
  {mux5.pin(7), {MCU::MUTE_8}},
 
  {mux10.pin(0), {MCU::MUTE_1}},
  {mux10.pin(1), {MCU::MUTE_2}},
  {mux10.pin(2), {MCU::MUTE_3}},
  {mux10.pin(3), {MCU::MUTE_4}},
  {mux10.pin(4), {MCU::MUTE_5}},
  {mux10.pin(5), {MCU::MUTE_6}},
  {mux10.pin(6), {MCU::MUTE_7}},
  {mux10.pin(7), {MCU::MUTE_8}},
};```
can it be done to send mcu value to my controler or mcu is just an input?
I got an error when I write it

**Comment by tttapa** [comment]
Sounds like you need `NoteButton` (output), not `NoteValue` (input). Have a look at the modules page of the documentation for the distinction between MIDI Input Elements and MIDI Output Elements.

**Reply by benwadub** [reply]
Yes I finally found that! Thanks
